src/main/reactjs/src/tests/cypress/integration/clerk_translator_details.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { APIEndpoints } from 'enums/api';
+import { onClerkHomePage } from 'tests/cypress/support/page-objects/clerkHomePage';
+import {
+  existingClerkTranslatorDetails,
+  onClerkTranslatorDetailsPage,
+} from 'tests/cypress/support/page-objects/clerkTranslatorDetailsPage';
+import { useFixedDate } from 'tests/cypress/support/utils/date';
+
+const fixedDateForTests = new Date('2022-01-17T12:35:00+0200');
+
+beforeEach(() => {
+  useFixedDate(fixedDateForTests);
+  cy.intercept(APIEndpoints.ClerkTranslator, {
+    fixture: 'clerk_translators_100.json',
+  });
+});
+
+const fakeTranslatorId = 1234567890;
+
+const expectTranslatorContactDetailsFields = () => {
+  Object.keys(existingClerkTranslatorDetails.contactDetails).forEach(
+    (field) => {
+      const uiField = onClerkTranslatorDetailsPage.contactDetailsField(field);
+      uiField.should(
+        'have.value',
+        existingClerkTranslatorDetails.contactDetails[field]
+      );
+      uiField.should('be.disabled');
+    }
+  );
+};
+
+const expectTranslatorAuthorisationDetails = () => {
+  existingClerkTranslatorDetails.authorisations.forEach((a) => {
+    const authorisationsRow =
+      onClerkTranslatorDetailsPage.authorisationDetailsRow(a.id);
+    authorisationsRow.should('contain.text', a.diaryNumber);
+  });
+};
```

Comment:
Siirtäisin noitakin page modelliin, jos onnistuu helposti, mutta saa olla täälläkin.

Reply:
Voisin pitää tässä tiedostossa; nää on nyt vähän fiksumpia kun noi käyttöliittymän assertiot on paremmin abstraktoitu pois.

Reply:
Jess, tehdään näin!